src/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -150,6 +151,7 @@
     <Compile Include="System\AttributeTests.netstandard.cs" />
     <Compile Include="CompareToTests.cs" />
     <Compile Include="DBNullTests.netstandard.cs" />
+    <Compile Include="DBNullTests.InvalidCast.cs" Condition="'$(TargetGroup)' != 'uap'" />
```

Comment:
you are right this should be conditioned to uapaot but we don't have a configuration for it. I'll add that.